es/criteria/bundle-policy-and-code.md

```diff
@@ -2,49 +2,49 @@
 * Proporcionar referencias y enlaces a los textos que apoyan las políticas.
-* Documentar la política en formatos que no sean ambiguos y que sean legibles por la máquina, como [Modelo y notación de procesos empresariales](https://en.wikipedia.org/wiki/Business_Process_Model_and_Notation), [Modelo y notación de decisiones](https://www.omg.org/dmn/) y [Notación del modelo de gestión de casos](https://www.omg.org/cmmn/).
-* Realiza un seguimiento de la política con [el mismo control de versiones](version-control-and-history.md) y la documentación utilizada para el seguimiento del código fuente.
-* Comprueba regularmente cómo ha cambiado el código no relacionado con la política en la codebase y si sigue coincidiendo con las [intenciones de la política](document-objectives.md).
+* Documentar la política en formatos que no sean ambiguos y que sean legibles por computador, como [Modelo y notación de procesos empresariales](https://en.wikipedia.org/wiki/Business_Process_Model_and_Notation), [Modelo y notación de decisiones](https://www.omg.org/dmn/) y [Notación del modelo de gestión de casos](https://www.omg.org/cmmn/).
+* Realizar seguimiento de la política con [el mismo control de versiones](version-control-and-history.md) y la documentación utilizada para el seguimiento del código fuente.
+* Verificar regularmente cómo ha cambiado el código no relacionado con la política en la codebase y si sigue coincidiendo con las [intenciones de la política](document-objectives.md).
 
 ## Profesionales de la dirección de equipos: qué necesitan hacer
 
-* Mantén a las personas responsables políticas, a profesionales del desarrolladores y del diseño involucrados y conectados a lo largo de todo el proceso de desarrollo.
-* Asegúrate de que las personas responsables políticas, los profesionales del desarrollo y del diseño trabajan con los mismos objetivos.
+* Mantener a las personas responsables de políticas, a profesionales del desarrolladores y del diseño involucrados y conectados a lo largo de todo el proceso de desarrollo.
+* Asegurarse de que las personas responsables de políticas, los profesionales del desarrollo y del diseño trabajan con los mismos objetivos.
 
 ## Profesionales del desarrollo de software y diseño: qué necesitan hacer
 
-* Familiarízate con la notación de modelado de procesos que utilizan las y los responsables políticos de tu organización y sé capaz de utilizarla.
-* Trabaja junto a responsables de las políticas para garantizar que no haya desajustes entre el código de las políticas y el código fuente.
-* Da información sobre cómo hacer que la documentación de las políticas sea más clara.
+* Familiarízarse con, y ser capaz de utilizar, la notación de modelado de procesos que utilizan las y los responsables de políticas de su organización.
+* Trabajar junto a las y los responsables de las políticas para garantizar que no haya incongruencias entre el código de las políticas y el código fuente.
```

Comment:
I like both the "Trabajar" and "incongruencias" suggestions. Not so much the articles